app/assets/javascripts/modules/ajax_save.js

```diff
@@ -117,15 +117,20 @@
 
       function showErrors (errors) {
         $.each(errors, function (errorKey, errorMessages) {
-          var errorElement = element.find('#edition_' + errorKey + '_input')
-          var $list = $('<ul class="help-block js-error"></ul>')
+          var errorElement = null
 
-          errorElement.addClass('has-error')
-          for (var j = 0, m = errorMessages.length; j < m; j++) {
-            $list.append('<li>' + errorMessages[j] + '</li>')
+          if (errorKey.endsWith('_availability')) {
+            errorElement = element.find('#edition_' + errorKey + '_attributes_alternative_url')
+          } else {
+            errorElement = element.find('#edition_' + errorKey)
           }
+          var parents = errorElement.parents('.form-group')
+          parents.addClass('has-error')
 
-          errorElement.append($list)
+          var list = parents.find('.error-block')
```

Comment:
Not all fields - no.  But all validated fields will.  That's the idea behind using partials for field types (e.g. `_validated_text_field.html.erb`) so that they have this same basic structure...

```
<div class="form-group">
  <div class="form-label">...</div>
  <div class="form-wrapper">...</div>
  <div class="help-block">...</div>
  <ul class="error-block">...</ul>
</div>
```

I'm thinking of introducing a `_validated_element.html.erb` partial if possible - that has this structure yet yields to the field specific partials as required.  The only thing that really needs to differ is the actual input element.  

Haven't tried doing this so far as I'm not quite sure how to actually do it yet.